api/swig/python/setup.py.in

```diff
@@ -0,0 +1,35 @@
+import setuptools.command.install
+import shutil
+from distutils.sysconfig import get_python_lib
+
+
+class CompiledLibInstall(setuptools.command.install.install):
+    """
+    Specialized install to install to python libs
+    """
+
+    def run(self):
+        """
+        Run method called by setup
+        :return:
+        """
+        # Get filenames from CMake variable
+        filenames = '${PYTHON_INSTALL_FILES}'.split(';')
```

Comment:
This generates not portable code (and it misses the BAG DLL on Windows): 

![notepad++_umMcTbyG7K](https://user-images.githubusercontent.com/2849257/192089529-4d86823c-9de9-4b8c-bf9c-e765631e05f3.png)

Reply:
Will address in version 2.0.2, see: https://github.com/OpenNavigationSurface/BAG/issues/69.